src/nvoptix.c

```diff
@@ -120,13 +122,55 @@ static BOOL load_nvoptix(void)
 
     #undef LOAD_FUNCPTR
 
+    if (callbacks_enabled())
+    {
+        pthread_rwlockattr_t attr;
+
+        if (pthread_rwlockattr_init(&attr))
+        {
+            ERR("Failed to initialize rwlockattr.\n");
+            goto fail;
+        }
+
+        if (pthread_rwlockattr_setpshared(&attr, PTHREAD_PROCESS_SHARED))
+        {
+            ERR("Failed to configure rwlockattr.\n");
+            goto fail;
+        }
+
+        if (pthread_rwlock_init(&callbacks_lock, &attr))
+        {
+            ERR("Failed to initialize rwlock.\n");
+            
```

Comment:
NIT: Seems to be an overabundance of spaces here :smile:

Reply:
Whoops. Should be fixed now.